datafusion/core/src/execution/mod.rs

```diff
@@ -15,30 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-//! This module contains the shared state available at different parts
```

Comment:
this was hard to find, so I moved the content on to the structs that were referenced